technologies/app/mlflow-server-1.20/mlflow-server-1.20/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.7.0
```

Comment:
Why not use the latest 3.7?
https://hub.docker.com/layers/python/library/python/3.7.12/images/sha256-458a687f85bb5e308cb50e9694322a5033e2091c6462eaa78a842a7df5f12fad?context=explore

Reply:
You can use this version https://hub.docker.com/layers/python/library/python/3.7.11-stretch/images/sha256-a1ee982556d76c1df300208cf82de78efb1b70908ae4e665dfc3fa2b467499f8?context=explore

Use debian version with openjdk8.